### DISCUSSION TITLE
Can't use Quarto Convert

### DISCUSSION THREAD

**Discussion by ghost** [quarto-dev/quarto-cli#6841] [Q&A]
### Description

For some reason when I try to use Quarto Convert with a VSCode file it keeps returning the same error over and over (image included below). I have tried to create empty test files and install/reinstall Quarto, VSCode, R, etc nothing has solved the problem. I am admittedly new to this but would appreciate some help nonetheless. 
<img width="699" alt="Screenshot 2023-09-15 at 1 37 23 AM" src="https://github.com/quarto-dev/quarto-cli/assets/136371900/c3ff3d3c-2ebb-4ab7-af21-3766d17f9b6d">

**Comment by mcanouil** [comment]
Note that the message states you have invalid YAML frontmatter in your file.

There are "Get Started" guides: <https://quarto.org/docs/get-started/> which show examples of valid files.
See also: <https://quarto.org/docs/tools/vscode-notebook.html#converting-notebooks>.

---

For more, consider sharing a small self-contained "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.
With only an error message without any context such as the environment (i.e., operating system, Quarto version, etc.), there is not much we can do to help.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````
